projects/v2/src/app/components/info/info.component.html

```diff
@@ -69,15 +69,15 @@
       <div *ngIf="info.entrezLink">
       <div class="pt-3"></div>
 
-      <label ><b>Entrez Link</b></label>
+      <label ><strong>Entrez Link</strong></label>
```

Comment:
Generalize this to an *ngFor over Object.items(extraLinks). The idea was that extra links could be provided (and doesn't need to be hard coded here).